create_container.sh

```diff
@@ -0,0 +1,8 @@
+
+# Run ONCE to create container
+
+#!/bin/bash
+docker build -t desktop-business-app .
+
+# Run the container in the background (detached mode)
+docker run -d --name desktop-business-app --net=host -e DISPLAY=$DISPLAY -v $(pwd):/application -w /application desktop-business-app tail -f /dev/null
```

Comment:
I placed commands "docker build" and "docker run" in script "create_container.sh" because you use them only once to create image and container. Commands for opening container are in script "run_container.sh" because you will use them repeteadly to enter existing container. 
I agree that in that case name "open_container" might be better then "run_container". It looks like it is just a matter of naming.

Reply:
All right, we can adjust naming later on, approved